moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_collision.cpp

```diff
@@ -58,21 +60,10 @@ static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.fix_start_st
 class FixStartStateCollision : public planning_request_adapter::PlanningRequestAdapter
 {
 public:
-  static const std::string DT_PARAM_NAME;
-  static const std::string JIGGLE_PARAM_NAME;
-  static const std::string ATTEMPTS_PARAM_NAME;
-
   void initialize(const rclcpp::Node::SharedPtr& node, const std::string& parameter_namespace) override
   {
-    node_ = node;
-    max_dt_offset_ = getParam(node_, LOGGER, parameter_namespace, DT_PARAM_NAME, 0.5);
-    jiggle_fraction_ = getParam(node_, LOGGER, parameter_namespace, JIGGLE_PARAM_NAME, 0.02);
-    sampling_attempts_ = getParam(node_, LOGGER, parameter_namespace, ATTEMPTS_PARAM_NAME, 100);
-    if (sampling_attempts_ < 1)
-    {
-      sampling_attempts_ = 1;
-      RCLCPP_WARN(LOGGER, "Param '%s' needs to be at least 1.", ATTEMPTS_PARAM_NAME.c_str());
-    }
+    param_listener_ =
+        std::make_shared<default_plan_request_adapter_parameters::ParamListener>(node, parameter_namespace);
```

Comment:
nit: `initialize` should take the node by const-value and then `std::move` it here.